test/integration_test/basic_test.go

```diff
@@ -52,46 +43,19 @@ var testStorageClusterBasicCases = []types.TestCase{
 			require.NoError(t, err)
 			cluster, ok := objects[0].(*corev1.StorageCluster)
 			require.True(t, ok)
-			cluster.Name = "custom-ns-shifted-port-all-components"
+			cluster.Name = "test-stc"
 			cluster.Namespace = "custom-namespace"
 			cluster.Spec.StartPort = func(val uint32) *uint32 { return &val }(17001)
 			return cluster
 		},
 		TestFunc: BasicInstallInCustomNamespace,
 	},
-	{
-		TestName:        "NodeAffinityLabels",
-		TestrailCaseIDs: []string{"C50962"},
-		TestSpec: ci_utils.CreateStorageClusterTestSpecFunc(&corev1.StorageCluster{
-			ObjectMeta: metav1.ObjectMeta{Name: "node-affinity-labels"},
-			Spec: corev1.StorageClusterSpec{
-				Placement: &corev1.PlacementSpec{
-					NodeAffinity: &v1.NodeAffinity{
-						RequiredDuringSchedulingIgnoredDuringExecution: &v1.NodeSelector{
-							NodeSelectorTerms: []v1.NodeSelectorTerm{
-								{
-									MatchExpressions: []v1.NodeSelectorRequirement{
-										{
-											Key:      labelKeySkipPX,
-											Operator: v1.NodeSelectorOpNotIn,
-											Values:   []string{ci_utils.LabelValueTrue},
-										},
-									},
-								},
-							},
-						},
-					},
-				},
-			},
-		}),
-		TestFunc: BasicInstallWithNodeAffinity,
-	},
 	{
 		TestName:        "BasicUpgradeStorageCluster",
 		TestrailCaseIDs: []string{"C50241"},
 		TestSpec: func(t *testing.T) interface{} {
 			return &corev1.StorageCluster{
-				ObjectMeta: metav1.ObjectMeta{Name: "upgrade-stc-test"},
+				ObjectMeta: metav1.ObjectMeta{Name: "test-stc"},
 			}
 		},
 		ShouldSkip: func(tc *types.TestCase) bool {
```

Comment:
nit: we can add a new field here e.g. `SkipUninstall`, so it's easier to tell which test case preserve the cluster.